secret.py

```diff
@@ -0,0 +1,9 @@
+USER = "rsharan"
```

Comment:
don't commit your secret.py file!!!!!!

this is a public repo, make sure you change your API key immediately!!!